style/tailwind.css

```diff
@@ -1,3 +1,141 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.markdown-content {
+  @apply bg-base-200 mt-4 p-5 rounded space-y-4;
+}
+
+.markdown-content h1 {
+  @apply text-2xl font-black;
+}
+
+.markdown-content h2 {
+  @apply text-2xl font-bold;
+}
+
+.markdown-content h3 {
+  @apply text-xl font-black;
+}
+
+.markdown-content h4 {
+  @apply text-xl font-bold;
+}
+
+.markdown-content h5 {
+  @apply text-lg font-black;
+}
+
+.markdown-content h6 {
+  @apply text-lg font-bold;
+}
+
+.markdown-content code {
+  @apply font-mono bg-neutral text-neutral-content p-1 rounded-md not-italic font-medium slashed-zero text-sm;
+}
+
+.markdown-content blockquote {
+  @apply ps-1.5 pe-7 py-2 bg-base-300 border-s-4 border-info font-light text-sm italic relative  rounded-se-md rounded-ee-md break-all;
+}
+
+.markdown-content blockquote code {
+  @apply text-xs font-normal;
+}
+
+.markdown-content blockquote strong {
+  @apply font-semibold;
+}
+
+.markdown-content blockquote em {
+  @apply not-italic;
+}
+
+.markdown-content blockquote:not(blockquote blockquote)::after {
+  content: " ";
+  mask-image: url("/icons.svg#css-quote");
+  -webkit-mask-image: url("/icons.svg#css-quote");
+  @apply bg-info block size-6 absolute end-1 bottom-1;
+}
```

Comment:
The `mask-image` approach lets me control the color of the icon being used from a CSS URL.